catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/ModalRegistry.view.js

```diff
@@ -131,6 +125,34 @@ function (ich,Marionette,Backbone,ConfigurationEdit,wreqr,_,$,Utils,Service,moda
                     });
             }
         },
+        urlChanged: function(evt) {
+            var url = evt.target.value;
+            var regExUrl;
+            if (url) {
+                regExUrl = this.extractHostnameFromUrl(url);
+            }
+            var hostname;
+            if (regExUrl) {
+                hostname = regExUrl;
+                if (hostname) {
```

Comment:
hostname is being to set to regExUrl which was already testing so this will always be true. The else will never be hit.